winter/Code/headless/headless.cpp

```diff
@@ -1,22 +1,27 @@
 #include "headless.h"
 int main() {
   printf(WELCOME_TEXT);
-  auto p = new palace;  // does sanity checks
   A_init_error_system();
   A_error_system->RegisterListener(EventType::kOnError, [](Event& ev) {
     printf(static_cast<ErrorMessage&>(ev).get_message().c_str());
   });
-  p->find_sourcemod_path();
-  if (p->sourcemodsPath.empty()) {
-    printf("We couldn't find your sourcemod folder. Please input an alternate sourcemods path.\n");
-  try_again:
-    std::string path;
+  palace* p;
+  try {
+    p = new palace;  // does sanity checks
+  }catch(std::runtime_error &e){
+    printf("We couldn't find your steam folder. Please input an alternate sourcemods path.\n");
+    try_again:
+      std::string path;
```

Comment:
Indentation around the label here is inconsistent.

Reply:
it's fine for now the whole codebase is a mess (will be fixed in another pr)